source/operations/external-iam/configure-ad-ldap-external-identity-management.rst

```diff
@@ -99,40 +101,76 @@ An AD/LDAP user with no assigned policy *and* with membership in groups with no
 
    .. _minio-external-identity-management-ad-ldap-configure:
 
+
    Procedure
    ---------
 
    1) Set the Active Directory / LDAP Configuration Settings
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   You can configure the AD/LDAP provider using either
-   environment variables *or* server runtime configuration settings. Both
-   methods require starting/restarting the MinIO deployment to apply changes. The
-   following tabs provide a quick reference of all required and optional
-   environment variables and configuration settings respectively:
+   Configure the AD/LDAP provider using one of the following:
+
+   * MinIO Client
+   * environment variables
+   * MinIO Console
+
+   All methods require starting/restarting the MinIO deployment to apply changes.
+
+   The following tabs provide a quick reference of the three LDAP configuration methods:
 
    .. tab-set::
 
+      .. tab-item:: MinIO Client
+
+         MinIO supports specifying the AD/LDAP provider settings using :mc:`mc idp ldap` commands.
+
+	 Confirm this:
+         For distributed deployments, the :mc:`mc idp ldap` command applies the configuration to all nodes in the deployment. 
+
+         The following example code sets *all* configuration settings related to configuring an AD/LDAP provider for external identity management.
+	 The minimum *required* settings are:
+
+         - :mc-conf:`identity_ldap server_addr <identity_ldap.server_addr>`
+         - :mc-conf:`identity_ldap lookup_bind_dn <identity_ldap.lookup_bind_dn>`
+         - :mc-conf:`identity_ldap lookup_bind_password <identity_ldap.lookup_bind_password>`
+         - :mc-conf:`identity_ldap user_dn_search_base_dn <identity_ldap.user_dn_search_base_dn>`
+         - :mc-conf:`identity_ldap user_dn_search_filter <identity_ldap.user_dn_search_filter>`
+
+        .. code-block:: shell
+           :class: copyable
+
+	   mc idp ldap add ALIAS                                                   \
+	      server_addr="ldaps.example.net:636"                                  \
+              lookup_bind_dn="CN=xxxxx,OU=xxxxx,OU=xxxxx,DC=example,DC=net"        \
+	      lookup_bind_password="xxxxxxxx"                                      \
+	      user_dn_search_base_dn="DC=example,DC=net"                           \
+	      user_dn_search_filter="(&(objectCategory=user)(sAMAccountName=%s))"  \
+	      group_search_filter= "(&(objectClass=group)(member=%d))"             \
+	      group_search_base_dn="ou=MinIO Users,dc=example,dc=net"              \
+	      server_insecure=on
+
+        For more complete documentation on these settings, see :mc:`mc idp ldap`.
+
+	.. admonition:: :mc:`mc idp ldap` recommended
+           :class: note
+
+           :mc:`mc idp ldap` offers additional features and improved validation over :mc-cmd:`mc admin config set` runtime configuration settings.
+           The :mc-conf:`identity_ldap` configuration key remains available for existing scripts and tools.
```

Comment:
Need to decide what to say about configuring with `identity_ldap`. I haven't seen that it's formally deprecated, but the newer `mc idp ldap` is preferred.

My understanding is doing it with the configuration key _should_ emit a warning. Not sure the status of that. See https://github.com/minio/minio/issues/17790

Reply:
My understanding is we prefer users utilize `mc idp ldap` as it is just less error prone and a bit easier to manage. It effectively sets the config settings under the hood.

I don't think we would ever deprecate the direct config route - but what we probably _should_ do is add a new tab to the LDAP (and OpenID ) pages that uses the MC CLI, then envvars, then config - where we state in the latter tabs that we recommend `mc idp <thing>`.

Reply:
Yes, `mc admin config set ALIAS identity_ldap ...` is not deprecated - they are just lower level commands that were used previously - they will be available. The newer `mc idp ldap` is higher level and does more LDAP validation (for e.g. it should have better error messages when something is misconfigured or not working).

On minio/minio#17790 the problem is that the config parameter name is not right `identity_ldap_server_insecure="on"` - this has a unnecessary `identity_ldap_` prefix and it should throw an error, but there is an issue with the parser that may need some larger changes. I am marking it as low priority for now, but will get around to fixing it eventually.

Reply:
Aha! It was specifically about that parameter, not anything with `identity_ldap` more generally. That makes sense. 

The follow-on PR will separate the two methods even more and hopefully reduce confusion.